packages/flutter/lib/src/material/expansion_panels.dart

```diff
@@ -0,0 +1,294 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'expand_icon.dart';
+import 'mergeable_material.dart';
+import 'theme.dart';
+
+const double _kPanelHeaderCollapsedHeight = 48.0;
+const double _kPanelHeaderExpandedHeight = 64.0;
+
+/// This callback gets called when [ExpansionPanel] with index [panelIndex] gets
```

Comment:
We've been documenting callback typedefs mostly consistently, like this:

/// Signature for the callback that's called when an ExpansionPanel is expanded or collapsed.
///
/// The position of the panel within an [ExpansionPanelList] is given by [panelIndex].

I hope I got the panelIndex part right. I don't think you need to explain what isExpanded means.